pkg/k8s/labels.go

```diff
@@ -363,69 +363,54 @@ const (
 	IdentityServiceAccountTokenPath = "/var/run/secrets/kubernetes.io/serviceaccount/token"
 
 	/*
-	 * Service mirror constants
+	 * Multicluster labels
 	 */
 
-	// SvcMirrorPrefix is the prefix common to all labels and annotations
-	// and types used by the service mirror component
-	SvcMirrorPrefix = "mirror.linkerd.io"
+	// MulticlusterAnnotationsPrefix is the prefix common to all labels and
+	// annotations and types used by the service mirror component
+	MulticlusterAnnotationsPrefix = "multicluster.linkerd.io"
 
 	// MirrorSecretType is the type of secret that is supposed to contain
 	// the access information for remote clusters.
-	MirrorSecretType = SvcMirrorPrefix + "/remote-kubeconfig"
+	MirrorSecretType = MulticlusterAnnotationsPrefix + "/remote-kubeconfig"
 
 	// DefaultExportedServiceSelector is the default label selector for exported
 	// services.
-	DefaultExportedServiceSelector = SvcMirrorPrefix + "/exported"
+	DefaultExportedServiceSelector = MulticlusterAnnotationsPrefix + "/exported"
```

Comment:
nit: WDYT about changing this to be `/export` -- this fits more with annotations like `inject`.